update

```diff
@@ -7,6 +7,7 @@ const path = require('path')
 const semver = require('semver')
 const ethUtil = require('ethereumjs-util')
 const swarmhash = require('swarmhash')
+const cp = require('child_process')
```

Comment:
This is not used that often so I'd use full name for readability:
```suggestion
const child_process = require('child_process')
```
`cp` reads like the command-line copying utility to me :)

Reply:
very simple.😉